frontend/src/v4/routes/viewerGui/viewerGui.component.tsx

```diff
@@ -73,6 +73,7 @@ interface IProps {
 	selectedTicket: ITicket | undefined;
 	treeNodesList: any;
 	ticketsCardView: TicketsCardViews;
+	isEditingGroups: boolean;
```

Comment:
I can't see any component using this

Reply:
in line 192

Reply:
Oh, right, my bad.

In that case, I saw that the `view` object that you get from Ticket context is either `Form` or `Groups`; but:

- Using `TicketsCardHooksSelectors.selectView()` is possible to determine if you are looking at the `Form` (if the value is not `List`, it's either the new ticket or the details of an existing one, so the form view).
- With the new `isEditingGroups` is also possible to determine if the form is group mode, equivalent to `Groups` from ticketContext.

As such, do we still need to keep the `view` in ticketContext?

Reply:
Yes. If you take a look the PR undoes that idea because that approach makes it prone to errors. Because the ticketview property is agnostic to wether is a new ticket or you are updating an existing ticket, it is better that navigation (switching between groups and no groups) is also agnostic. Otherwise it has to know if you are going from a new ticket or an already existing one. Bottom line, that is an idea that already failed.

Reply:
I'll create a ticket for the pin bug